ui/src/tooltip.tsx

```diff
@@ -45,21 +47,23 @@ const
 
 export const
   XToolTip = ({ children, content, showIcon = true, expand = true }: {
-    children: React.ReactChild,
+    children: React.ReactElement,
     content?: S,
     showIcon?: B,
     expand?: B
   }) => {
     if (!content) return <>{children}</>
 
-    const tooltipProps: Fluent.ITooltipProps = { onRenderContent: () => <div><Markdown source={content} /></div> }
+    const
+      isFullHeight = children.props.model?.height === '1',
+      tooltipProps: Fluent.ITooltipProps = { onRenderContent: () => <div><Markdown source={content} /></div> }
```

Comment:
Is the surrounding `<div>` tag necessary?

Reply:
Apparently not, but additional testing needs to be done to ensure there is no reason why it was originally implemented with a `<div>` tag.

Reply:
After exhausting testing it seems not to be causing any issues, therefore I am removing it.